web/src/components/tasks/business-formation/business/BusinessStep.test.tsx

```diff
@@ -130,13 +130,13 @@ describe("Formation - BusinessStep", () => {
     const page = await getPageHelper(
       {
         legalStructureId: "limited-liability-company",
-        municipality: generateMunicipality({ displayName: "Newark", name: "Newark" }),
```

Comment:
If we're deleting this param, let's update the functions we're calling and remove any logic around it. There's no sense in hanging onto code that's no longer being used, that's why it's a reminder in our PR checklist.

Reply:
Updated

Reply:
Updated